src/main.rs

```diff
@@ -808,6 +896,16 @@ fn main() {
         return;
     }
 
+    if let Err(e) = expand_fdtable().map_err(Error::ExpandFdTable) {
+        match e {
+            Error::ExpandFdTable(FdTableError::Close(_)) => {
+                eprintln!("{e}");
```

Comment:
nit: I think both of these should have a prefix like "Error expanding FD table:" - on their own the error messages may not be helpful.